bin/coreos-install

```diff
@@ -355,7 +359,7 @@ if [[ "${VERSION_ID}" =~ ^(alpha|beta|stable)$ ]]; then
 fi
 
 if [[ -z "${BASE_URL}" ]]; then
-    BASE_URL="https://${CHANNEL_ID}.release.core-os.net/amd64-usr"
+    BASE_URL="https://${CHANNEL_ID}.release.core-os.net/${ARCH}-usr"
```

Comment:
Let's call this BOARD instead, with the default value being "amd64-usr" (this is the scheme we use in the rest of our build tools). This would allow us to offer different OS layouts for a single architecture in the future.